src/features/documentation.ts

```diff
@@ -2,29 +2,30 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-'use strict';
-
 
-const _regExp = /<(\S*?).*?>((.|\r|\n)*?)<\/\1>/;
+'use strict';
 
-/**
- * remove xml-tags from string
- */
-export function plain(doc: string): string {
+const summaryStartTag = /<summary>/i;
+const summaryEndTag = /<\/summary>/i;
 
-	if (!doc) {
-		return doc;
+export function extractSummaryText(xmlDocComment: string): string {
```

Comment:
I realize this may be opening a can of worms, but do we have any facilities for unit testing this stuff?